sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -908,15 +904,18 @@ def _get_container_image_tag():
   return base_version
 
 
-def get_default_container_image_for_current_sdk(use_fnapi):
+def get_container_image_from_options(pipeline_options):
   """For internal use only; no backwards-compatibility guarantees.
 
     Args:
-      use_fnapi (bool): True, if pipeline is using FnAPI, False otherwise.
+      pipeline_options (PipelineOptions): A container for pipeline options.
 
     Returns:
-      str: Google Cloud Dataflow container image for remote execution.
+      str: Container image for remote execution.
     """
+  if pipeline_options.As(WorkerOptions).worker_harness_container_image:
```

Comment:
I think correct syntax is `pipeline_options.view_as(WorkerOptions)`. Does current syntax work?

Reply:
Fixed.